test/integration_test_helper.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module KubernetesDeploy
+  class IntegrationTest < KubernetesDeploy::TestCase
+    include KubeclientHelper
+    include FixtureDeployHelper
+
+    TestProvisioner.prepare_cluster
+
+    if ENV["PARALLELIZE_ME"]
+      puts "Running tests in parallel!"
+      parallelize_me!
+    end
+
+    def run
+      WebMock.allow_net_connect!
```

Comment:
This is never turned off again? If you always want this on, I'd probably move this to `test_helper` or something.

Reply:
I want it on for all integration tests and off for all unit tests. Moving to the superclass's `run` in test_helper to make it more obvious what is happening.